keg_elements/db/mixins.py

```diff
@@ -298,6 +298,16 @@ class DefaultMixin(DefaultColsMixin, MethodsMixin):
     pass
 
 
+class InternalDefaultMixin(DefaultMixin):
```

Comment:
I'd put this in the default implementation and not require changing the class hierarchy.  Especially since it's a no-op by default.